ui/shared/components/panel/search/constants.js

```diff
@@ -375,6 +375,114 @@ export const LOCATION_FIELDS = [
   },
 ]
 
+export const IN_SILICO_FIELDS = [
+  {
+    name: 'cadd',
+    label: 'CADD',
+    labelHelp: 'Enter a numeric value for Cadd prediction. During filtering, some variants might be left out, since their annotation value might be rounded up or down from their original value',
```

Comment:
The `During filtering, some variants might be left out, since their annotation value might be rounded up or down from their original value` should be removed from all of these. This language was never discussed or approved by me